packages/client/src/components/Actions/components/TokenView.js

```diff
@@ -0,0 +1,66 @@
+import { useClipboard, useContacts } from "hooks";
+import Svg from "library/Svg";
+import { getSafeContactName, getTokenMeta } from "utils";
+import ProposedDateView from "./ProposedDateView";
+
+const TokenView = ({ actionView, safeData }) => {
+  const { recipient, vaultId, tokenAmount, timestamp } = actionView;
+  const { name: safeName, address: safeAddress } = safeData;
+
+  const { displayName, tokenType } = getTokenMeta(vaultId) || {};
+
+  const clipboard = useClipboard();
+  const { contacts } = useContacts(safeAddress);
+
+  return (
+    <>
+      <div className="pl-4 mb-5">
+        <span className="columns">Amount</span>
+        <span className="columns is-size-2 is-family-monospace has-text-black">
+          {Number(tokenAmount)}
```

Comment:
```suggestion
          {Number(tokenAmount).toLocaleString()}
```
This will make sure the number gets delaminated with comma